### PR TITLE
Add LOBID fetcher

### PR DESCRIPTION
**Pull request by aqurilla**
Fixes koppor#386 by adding the LOBID fetcher (https://lobid.org/resources/api) 

### Mandatory checks
- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Yeah! Thanks for tackling this issue. I think if we need some support, we can probably also ask at their repo:
https://github.com/hbz/lobid-resources

**Comment by aqurilla** [comment]
@Siedlerchr Sounds good!

**Comment by Siedlerchr** [comment]
Is your PR ready for review? Looks already very good

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We added support for customizating the citation command (e.g., `[@key1,@key2]`) when [pushing to external applications](https://docs.jabref.org/cite/pushtoapplications). [#10133](https://github.com/JabRef/jabref/issues/10133)
 - We added an integrity check for more special characters. [#8712](https://github.com/JabRef/jabref/issues/8712)
 - We added protected terms described as "Computer science". [#10222](https://github.com/JabRef/jabref/pull/10222)
+- We added a fetcher for [LOBID](https://lobid.org/resources/api) resources. [#386](https://github.com/koppor/jabref/issues/386)
```
Comment (Siedlerchr) [inline CHANGELOG.md:19]:
```suggestion
- We added a fetcher for [LOBID](https://lobid.org/resources/api) resources. [koppor#386](https://github.com/koppor/jabref/issues/386)
```

**Review by Siedlerchr — DISMISSED** [review]

**Comment by aqurilla** [comment]
@Siedlerchr yes this is ready for review!

**Review by Siedlerchr — APPROVED** [review]

**Review by koppor — APPROVED** [review]

**Pull request merged**